Assignment_2/Index.html

```diff
@@ -127,12 +127,16 @@
         dataType: 'json', // added data type
         success: function(res) {
             console.log(res);
-           var result=[];
-            for (x of res.game_indices)
-            {
-                result.push(x.version.name);
+           let result=res.game_indices.map(iter)
```

Comment:
You can simplify like this `let result=res.game_indices.map((game) => game.version.name)`
So everything will be in one line. 
But this is also fine 👍

Reply:
Hi Althaf, I tried but I think the res.indices is an array and hence it is showing undefined.